src/main/java/org/cyclops/energeticsheep/entity/EntityEnergeticSheepConfig.java

```diff
@@ -107,7 +115,7 @@ public int getForegroundEggColor() {
     public void onRegistered() {
         super.onRegistered();
         for (Biome biome : Biome.REGISTRY) {
```

Comment:
Make sure to skip the loop if `spawnChance <= 0`. No need to iterate all biomes when the weight is zero in any case.

Reply:
Done